js/scripts/util.js

```diff
@@ -41,6 +41,29 @@ const FONT = { // NOTE: the "vt323" part below is not what's assigning the vt323
     VT323_HEADER: "24px vt323"
 };
 
+/**
+ * The game's sound effects.
+ */
+const SFX = {
+    JUMP1: {path: "./sfx/jump1.mp3", volume: 0.2},
```

Comment:
Should all of these volumes be different? Maybe they should be and this might be great, it would be cool if we can update all of the sfx volumes together after you are done mixing/mastering the individuals sfx sounds to be the right volumes. Nice work! @NathanHinthorne